java-checks/src/main/java/org/sonar/java/checks/LazyArgEvaluationCheck.java

```diff
@@ -185,6 +187,7 @@ private static String[] testMethodNames(String[] lowerCaseNames) {
 
   private JavaFileScannerContext context;
   private Deque<Tree> treeStack = new ArrayDeque<>();
+  private boolean insideLevelTestReturnStatementFound = false;
```

Comment:
What do you think of renaming this variable to `foundReturnStatementInsideLevelTest`?